iron-list.html

```diff
@@ -593,10 +589,14 @@
     attached: function() {
       this.updateViewportBoundaries();
       this._render();
+      // `iron-resize` is fired when the list is attached if the event is added
+      // before attached causing unnecessary work.
+      this.listen(this, 'iron-resize', '_resizeHandler');

```

Comment:
How about just keep the listeners object and in `_resizeHandler` we add a check for `this.isAttached`?